content/_img-sources/ecosystem.uxf

```diff
@@ -0,0 +1,411 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Reviewers can ignore this file. This is just the [umlet source file](https://www.umlet.com/) for the `ecosystem.png` image below.